content/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/configuration.md

```diff
@@ -58,8 +99,9 @@ ly_add_target(
         *.AutoComponent.xml,AutoComponentTypes_Source.jinja,$path/AutoComponentTypes.cpp
 )
 ```
-
-Next, create the `<projectname>_autogen_files.cmake` file. The contents of this file add the source templates for [autocomponents](./autocomponents) to the project build.
+<a id="adding_autogen_file"></a>
+### Adding AutoGen CMake file
+Next, create a new file called `<projectname>_autogen_files.cmake` and make it a sibling of `CMakeList.txt`. Example: `<ProjectName>/Code/<projectname>_autogen_files.cmake`. The contents of this file add the source templates for [autocomponents](./autocomponents) to the project build.
```

Comment:
nit: "Make a sibling of CmakeList.txt" is over complicating things. Maybe "Next, create a new file called `<projectname>_autogen_files.cmake` and place it in the Project's code folder. For example `<ProjectName>/Code/<projectname>_autogen_files.cmake`."